configs/templates/hpc-dev-v1/spack.yaml

```diff
@@ -59,10 +59,10 @@ spack:
     fftw:
       version: [3.3.10]
     fiat:
-      version: [1.0.0]
+      version: [1.1.0]
     fms:
-      version: [2022.02]
-      variants: +64bit +quad_precision +gfs_phys +openmp +fpic
+      version: [2022.04]
+      variants: +64bit +quad_precision +gfs_phys +openmp +fpic constants=GFS
```

Comment:
For my education: what does `constants=GFS` add?

Reply:
The new fms (since 2022.03) has different sets of physical constants built in, one for GFDL, one for GEOS, one for GFS. Sigh. Very soon we'll need to support 2-3 different versions, one for each set of constants, and each of them built with both 32bit and 64bit. We should consider adding the constants identifier to the module name when we get to that point.